control/Controller.php

```diff
@@ -69,6 +69,13 @@ class Controller extends RequestHandler implements TemplateGlobalProvider {
 	 * @uses BasicAuth::requireLogin()
 	 */
 	public function init() {
```

Comment:
How about making the method protected? :D

You can put the deprecation notice inside __call